spec/models/user_spec.rb

```diff
@@ -4,6 +4,7 @@
 
   describe "relationships" do
     it {should have_many :orders}
+    # it {should belong_to :merchant}
```

Comment:
I think leaving this out is the correct choice. I think it would make it so that all users must belong to merchants.